ranger/data/scope.sh

```diff
@@ -87,7 +87,15 @@ handle_extension() {
             ## Preview as text conversion
             odt2txt "${FILE_PATH}" && exit 5
             ## Preview as markdown conversion
-            pandoc -s -t markdown -- "${FILE_PATH}" && exit 5
+            ulimit -v 500000  # Limit to 500MB
+            timeout 10s pandoc -s -t markdown -- "${FILE_PATH}" && exit 5
```

Comment:
`timeout` is part of Coreutils so it's not available on macOS and I'm not sure it's part of BusyBox.